### PR TITLE
[Logs] Logs glossary revision

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

**Review by victor-perov — APPROVED** [review]

File: content/logs/reference/glossary.md [hunk 1]
```diff
@@ -12,73 +12,73 @@ weight: 132
 
 #### Request
 
-A record of each request that has been processed through the Cloudflare network.
+A request is a communicative message that is transmitted between the client, or web browser, to a server. Each request that has been processed through the Cloudflare network generates a record
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: What do you mean by `communicative message`? Also, is there something missing at the end of the sentence? Should there be something else after `generates a record` or just a `.`?

File: content/logs/reference/glossary.md [hunk 2]
```diff
@@ -12,73 +12,73 @@ weight: 132
...
 
 #### Static content
 
-Website content that can be delivered to an end user directly from cache, without hitting the origin server.
+Static content is any file that is stored in a server and is the same every time it is delivered to users. HTML files and images are examples of this kind of content. This type of website content can be delivered directly from cache, without hitting the origin server.
 
 #### Dynamic content
 
-Website content that has to be fetched from the origin server.
+Dynamic content is content that changes based on factors specific to the user such as time of visit, location, and device. News websites or social media are examples of this type of content. This type of website content every time it is requested has to be fetched from the origin server.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]:
```suggestion
Dynamic content is content that changes based on factors specific to the user such as time of visit, location, and device. News websites or social media are examples of this type of content. For this type of website, content has to be fetched from the origin server every time it is requested.
```

File: content/logs/reference/glossary.md [hunk 3]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
+#### Bad browser score
 
-This value results from the Cloudflare **Browser Integrity Check (BIC)** feature. It indicates that the source of the request was not legitimate or that the request itself was malicious. Users would see a Cloudflare error `1010` page in their browser. Cloudflare's **Browser Integrity Check** looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
+The Bad browser score indicates that the source of the request was not legitimate or that the request itself was malicious. This value results from the [Cloudflare Browser Integrity Check (BIC)](https://support.cloudflare.com/hc/en-us/articles/200170086-Understanding-the-Cloudflare-Browser-Integrity-Check) feature. Users would see a Cloudflare error `1010` page in their browser. Cloudflare’s Browser Integrity Check looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]:
```suggestion
The Bad browser score indicates that the source of the request was not legitimate or that the request itself was malicious. This value results from the [Cloudflare Browser Integrity Check (BIC)](https://support.cloudflare.com/hc/articles/200170086) feature. Users would see a Cloudflare error `1010` page in their browser. Cloudflare’s Browser Integrity Check looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
```

File: content/logs/reference/glossary.md [hunk 4]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
-#### Blocked hotlink
+#### Blocked hotlink score
 
-This value results from the Cloudflare **Hotlink Protection** feature. **Hotlink Protection** ensures that other sites cannot use your bandwidth by building pages that link to images hosted on your server. Cloudflare customers can turn this feature on and off.
+The Blocked hotlink value results from the [Cloudflare Hotlink Protection](https://support.cloudflare.com/hc/en-us/articles/200170026-Understanding-Cloudflare-Hotlink-Protection) feature. Hotlink Protection ensures that other sites cannot use your bandwidth by building pages that link to images hosted on your server. Cloudflare customers can turn this feature on and off.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]:
```suggestion
The Blocked hotlink value results from the [Cloudflare Hotlink Protection](https://support.cloudflare.com/hc/articles/200170026) feature. Hotlink Protection ensures that other sites cannot use your bandwidth by building pages that link to images hosted on your server. Cloudflare customers can turn this feature on and off.
```

File: content/logs/reference/glossary.md [hunk 5]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
-Challenge based on user configurations set for visitor’s IP in either the Web Application Firewall (WAF) or Cloudflare Firewall Rules.
+​​New Captcha (user) is a challenge based on user configurations set for visitor’s IP in either the [Web Application Firewall (WAF)](/waf/) or [Cloudflare Firewall Rules](/firewall/cf-firewall-rules/).
 
 ### Traffic
 
 #### NoRecord
 
-No record was found about this request/IP to classify it somewhere else.
+​​The value NoRecord indicates that no record was found about a request/IP, so it cannot be classified.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: Should NoRecord be monospaced?

File: content/logs/reference/glossary.md [hunk 6]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
 #### Allowlist
 
-The request has been sent from an allowlisted IP address.
+The value Allowlist indicates that the request was sent from an allowlisted IP address.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: Same question for Allowlist and monospace. Also, should it be `AllowList`?

File: content/logs/reference/glossary.md [hunk 7]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
 #### SearchEngine
 
-The search engine used to send the request.
+The value SearchEngine indicates the search engine used to send the request.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: Monospace for SearchEngine?

File: content/logs/reference/glossary.md [hunk 8]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
 #### BadHost
 
-A bad host header or no host header was used.
+The value BadHost indicates that a bad host header or no host header was used in the request.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: Monospace for BadHost?

File: content/logs/reference/glossary.md [hunk 9]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
 #### Tor
 
-The request was made using a Tor browser.
-
-#### BadIP
-
-A request coming from an IP address that is not trusted by Cloudflare based on the **Threat Score**.
+The value Tor indicates that the request was made using a Tor browser.
```
Comment (marciocloudflare) [inline content/logs/reference/glossary.md]: Monospace for Tor value?

**Review by marciocloudflare — APPROVED** [review]

File: content/logs/reference/glossary.md (hunk 1 above)
Comment (angelampcosta) [reply content/logs/reference/glossary.md]: Good point! It sounds a bit redundant, I think `message` is enough.

File: content/logs/reference/glossary.md [hunk 10]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
 
 #### Total Threats Stopped
 
-Indicates the number of suspicious and bad requests aimed at your site.
+Total Threats Stopped indicates the number of suspicious and bad requests aimed at your site.
```
Comment (soheiokamoto) [inline content/logs/reference/glossary.md]: nit, but for consistency with others, maybe:
```suggestion
Total threats stopped indicates the number of suspicious and bad requests aimed at your site.
```

File: content/logs/reference/glossary.md [hunk 11]
```diff
@@ -89,86 +89,82 @@ The security levels and the challenge display criteria are:
...
 
-#### Bad browser
+#### Bad browser score
 
-This value results from the Cloudflare **Browser Integrity Check (BIC)** feature. It indicates that the source of the request was not legitimate or that the request itself was malicious. Users would see a Cloudflare error `1010` page in their browser. Cloudflare's **Browser Integrity Check** looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
+The Bad browser score indicates that the source of the request was not legitimate or that the request itself was malicious. This value results from the [Cloudflare Browser Integrity Check (BIC)](https://support.cloudflare.com/hc/articles/200170086) feature. Users would see a Cloudflare error `1010` page in their browser. Cloudflare’s Browser Integrity Check looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
```
Comment (soheiokamoto) [inline content/logs/reference/glossary.md]: nit, but for consistency with others, maybe:
```suggestion
The bad browser score indicates that the source of the request was not legitimate or that the request itself was malicious. This value results from the [Cloudflare Browser Integrity Check (BIC)](https://support.cloudflare.com/hc/articles/200170086) feature. Users would see a Cloudflare error `1010` page in their browser. Cloudflare’s Browser Integrity Check looks for common HTTP headers used most often by spammers, and blocks access to your page. It also challenges visitors that do not have a user agent or use a non-standard user agent (commonly used by bots, crawlers, or visitors).
```

**Pull request merged**